pkg/daemon/multus/client-daemonset.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: multus-validation-test-client-{{ .ClientID }}
+  labels:
+    app: multus-validation-test-client
+    app.kubernetes.io/name: "client"
+    app.kubernetes.io/instance: "client-{{ .ClientID }}"
+    app.kubernetes.io/component: "client"
+    app.kubernetes.io/part-of: "multus-validation-test"
+    app.kubernetes.io/managed-by: "kubectl-rook-ceph"
```

Comment:
Managed by the krew plugin?

Reply:
maybe in future we can add with krew since we are close to go transition.

Reply:
but currently, I don't think `kubectl-rook-ceph` is correct.

Reply:
Good point. It seems to me that `rook` might imply it's managed by the operator. What about `rook-cli`?